src/components/EditableTable.tsx

```diff
@@ -93,7 +99,7 @@ const EditableTable = ({
         >
           <div style={{ width: "100%", height: "100%" }}>
             {editingKey !== row.key ? (
-              transformedInput
+              projectedValue
```

Comment:
On a local chain, the bundle cost was back to its 100x price and I used cosgov to fix it. It said "no values have changed" or some such. This design where I have to hit save next to individual fields is not exactly smooth.

Orthogonal to this PR, I suppose.

Reply:
Also, I'm not sure whether I was using cosgov.org or this PR for that interaction.